datarepo-clienttests/src/main/java/testscripts/IngestFile.java

```diff
@@ -94,13 +89,15 @@ public void userJourney(ApiClient apiClient) throws Exception {
         repositoryApi.bulkFileLoadArray(datasetSummaryModel.getId(), fileLoadModelArray);
 
     ingestFileJobResponse = DataRepoUtils.waitForJobToFinish(repositoryApi, ingestFileJobResponse);
-    FileModel fileModel =
-        DataRepoUtils.expectJobSuccess(repositoryApi, ingestFileJobResponse, FileModel.class);
+
+    BulkLoadArrayResultModel bulkLoadArrayResultModel =
+        DataRepoUtils.expectJobSuccess(repositoryApi, ingestFileJobResponse, BulkLoadArrayResultModel.class);
+    BulkLoadFileResultModel fileInfo = bulkLoadArrayResultModel.getLoadFileResults().get(0);
+
     logger.debug(
-        "Successfully ingested file: path = {}, id = {}, size = {}",
-        fileModel.getPath(),
-        fileModel.getFileId(),
-        fileModel.getSize());
+        "Successfully ingested file: path = {}, id = {}",
+        fileInfo.getSourcePath(),
+        fileInfo.getFileId());
```

Comment:
I think you mentioned before you were trying to get the size out. Just double checking if that's still a value you're interested in pulling out.

Reply:
I just didn't like getting rid of metadata---but I dont see a way to pull this out in the new format